CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## 2.0.1-beta
+  - Improve testability
+    - Open classes for mockable
+    - Support for overloading 
```

Comment:
What does this mean?

Reply:
The support for mocking/verifying overloaded methods in the navigator. I'll be more specific